articles/step-by-step-on-how-to-user-nodejs-repl-tutorial/index.md

```diff
@@ -0,0 +1,208 @@
+### Introduction
+Node.js `Read-Evaluate-Print-Loop`(REPL) is an easy-to-use command-line tool for processing Node.js expressions.
+It captures the user's JavaScript code inputs, interprets and evaluates the result of this code, displays the result to the screen, and repeats the process till the user quits the shell.  
```

Comment:
@srishilesh, this paragraph here, says ...."it captures (reading), evaluate, display(print) and repeat(loop) what's it missing?.".